namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "formbuilder_ecr_credentials" {
```

Comment:
In `formbuilder-repos`, we're following the naming convention, `ecr-repo-fb-{app}`

	module "ecr-repo-formbuilder-product-page" ?
        module "ecr-repo-fb-product-page" ?

`formbuilder_ecr_credentials` sounds like a repo for all our credentials

Reply:
Changed the module name